src/ext/java/org/opentripplanner/ext/transmodelapi/model/framework/PassThroughPointInputType.java

```diff
@@ -0,0 +1,37 @@
+package org.opentripplanner.ext.transmodelapi.model.framework;
+
+import graphql.Scalars;
+import graphql.schema.GraphQLInputObjectField;
+import graphql.schema.GraphQLInputObjectType;
+import graphql.schema.GraphQLList;
+import graphql.schema.GraphQLNonNull;
+
+public class PassThroughPointInputType {
+
+  public static final GraphQLInputObjectType INPUT_TYPE = GraphQLInputObjectType
+    .newInputObject()
+    .name("PassThroughPoint")
+    .description("Defines one pass-through point which the journey must pass by.")
```

Comment:
Again, I find the passing-by/passing-through mismatch a little jarring.

Reply:
Fixed as well.